commands/publish/index.js

```diff
@@ -101,6 +101,20 @@ class PublishCommand extends Command {
       registry: this.options.registry,
     });
 
+    if (this.options["registry-scope"]) {
+      const registryScope = Array.isArray(this.options["registry-scope"])
```

Comment:
Where is this documented that it can accept more than one value? How does one discriminate?